polaris-pipeline/Common/Dto/Response/Case/DefendantsAndChargesListDto.cs

```diff
@@ -1,29 +1,30 @@
 using System.Collections.Generic;
 using System.Text.Json.Serialization;
-using Common.Dto.Response.Document.FeatureFlags;
+//using Common.Dto.Response.Document.FeatureFlags;
 using Newtonsoft.Json;
 
 namespace Common.Dto.Response.Case
 {
-    public class DefendantsAndChargesListDto
+    public class DefendantsAndChargesListDto : DefendantsAndChargesListCoreDto
```

Comment:
Base class introduced that has only the properties required for the coordinator/tracker.  We then type variables as the base class rather than the super class so. This means much smaller serialised JSONs are generated in the durable activity payloads.